webviewer/src/components/Areas/Sidebar/CreateRequest/CreateRequest.jsx

```diff
@@ -70,13 +70,15 @@ export const CreateRequest = ({ areas, currentArea }) => {
   };
 
   useEffect(() => {
-    const canSave = !notebook.period_required || (startDate && endDate);
+    const canSave =
+      (!notebook.period_required || (startDate && endDate)) &&
+      (!notebook.additional_parameter || Boolean(additionalParameterValue));
```

Comment:
what is the value of additionalParameterValue could be? 

you don't need Boolean wrapper. in case of you need even checking the value you can use !!additionalParameterValue instead of Boolean(additionalParameterValue)

Reply:
Updated with `!!additionalParameterValue`